developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md

```diff
@@ -11,6 +11,16 @@ Since you are already running [a validator node and a validator fullnode](node-r
 
 This page explains how to make this swap, which largely amounts to switching out files and configuration settings between the two nodes. For a community-provided version of this document for Docker setup, see [Failover and migrate Validator Nodes for less downtime](https://forum.aptoslabs.com/t/failover-and-migrate-validator-nodes-for-less-downtime/144846).
 
+## Secure
+
+Do not run multiple VFNs with the same [network identity](../../identity-and-configuration.md) and connect them to the validator using the `vfn` network, as this may cause issues with node metrics and telemetry. Nevertheless, we realize you may need access to REST APIs for building without any rate limits.
```

Comment:
nit: I'd maybe rephrase to something like:
```
If you wish to run multiple VFNs, e.g., to expose REST APIs for building products without any rate
limits, follow the instructions below. Note that it is not currently recommended to run multiple VFNs
with the same [network identity](../../identity-and-configuration.md) and connect them to the
validator using the `vfn` network, as this may cause issues with node metrics and telemetry.
```

Reply:
Done!